plat/intel/soc/stratix10/platform.mk

```diff
@@ -44,7 +44,9 @@ BL2_SOURCES     +=	\
 		lib/cpus/aarch64/cortex_a53.S				\
 		plat/intel/soc/stratix10/stratix10_image_load.c		\
 		plat/intel/soc/stratix10/soc/s10_system_manager.c	\
-                common/desc_image_load.c
+                common/desc_image_load.c				\
```

Comment:
nit: use tabs instead of spaces, like the other lines

Reply:
Done